code/modules/mob/mob_defines.dm

```diff
@@ -98,6 +98,9 @@
 
 	/// How many ticks this mob has been over reating
 	var/overeatduration = 0		// How long this guy is overeating //Carbon
+	
+	// if is above 0, supermatter energy ball tesla zap can gib this mob
+	var/mob_siemens_coeff_amount = 1
```

Comment:
What is this meant to accomplish?

Reply:
it is used in human_defense.dm `mob_siemens_coeff_amount` is equal to `siemens_coeff` so i can check if the mob is immue to tesla shocks to prevent the supermatter energy ball from gibbing the mob in energy_ball.dm line 337

Reply:
Can this not use the same checks we use for insulated?

Reply:
can't really because `siemens_coeff` is on item

Reply:
and siemens_coeff is for electrocute_act()
while mob_siemens_coeff_amount is for tesla_zap()

Reply:
ill attempt

Reply:
but this will change more files